alert-rules.md

```diff
@@ -621,7 +621,7 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
 * 处理方法：
 
     1. 一般是因为并行 GC 开的太高了，可以适当降低 GC 并行度。你需要先确认 GC 失败是由于服务器繁忙导致的。
```

Comment:
The `TiDB_tikvclient_gc_action_fail` section can be deleted since central GC is totally removed since 5.0

Reply:
Ok, it is now deleted from both the CN and EN PRs.